### Directory content

The script simulate_gene_expression_profiles_giotto.Rmd was used to simulate a scRNAseq dataset using the Giotto package https://github.com/RubD/Giotto. This script requires the files cortex_svz_expression.txt, cortex_svz_centroids_coord.txt & cortex_svz_centroids_annot.txt from the brain cortex seqFISH+ dataset https://rubd.github.io/Giotto_site/articles/mouse_seqFISH_cortex_200914.html. Based on these files, we used altered giotto functions in order to simulate gene expression levels. The function simulateOneGenePatternGiottoObject2 was used to simulate the gene expression levels while the function simulateOneGenePatternGiottoObject3 was used to specify the positions and grouping of the simulated cells (pardon the naming). This script will create a dataset of 42 simulated genes expressions over 422 cells. Then the script Furhter_improve_simulation_dataset.Rmd can be used to construct a dataset of 84 genes with alternating gene expressions over 2 groups. This script requires the previously constructed dataset and locations/grouping of the cells in the tissue. Furthermore, this script also require a file in which the gene names of 42 unique ligand-receptor interactions are described (isolated_genes.txt). The resulting simulated gene expression (simulated_gene_expression.txt) and simulated cell positions (simulated_cell_positions.txt) were stored on the google drive.  

